HISTORY.rst

```diff
@@ -12,7 +12,9 @@ History
 
 * Cleanup of the first release regarding naming, authors and docs.
 
-0.2.0 (2019-11-30)
+0.2.0 (2020-05-17)
 ------------------
 
 * Added functionality to use meta data for displaying descriptive statistics names and enum values
+* Added functionality to display the units of a statistic along with the numerical value.
+* Internally split the meta data extraction into technichal meta data and meta data about the statistics. Implmented new defaults for the statistics meta data in order to account for changes in the datenguide API.
```

Comment:
```suggestion
* Internally split the meta data extraction into technical meta data and meta data about the statistics. Implemented new defaults for the statistics meta data in order to account for changes in the datenguide API.
```

Adding simple changes like this as suggestions helps to commit the change directly inside the browser, for convenience.